composites/Plugin/ContentAnalysis/reducers/tests/overallScoreReducerTest.js

```diff
@@ -0,0 +1,119 @@
+import { SET_OVERALL_SCORE_SEO, SET_OVERALL_SCORE_READABILITY } from "../../actions/contentAnalysis";
+import { overallScoreReducer } from "../contentAnalysis/overallScoreReducer";
+
+describe( "SET_OVERALL_SCORE for seo action", () => {
+	it( "sets overall score for an seo analysis result", () => {
+		const state = {};
+		let firstKeyword = "thisIsMyFirstKeyword";
```

Comment:
Same as above: I appreciate not wanting to hard-code in the object, but adding the `firstKeyword` and `"thisIsMyFirstKeyword"` complicates the CR at least 😜 . 

It's a difference in style I guess, I'll leave it up to you.